tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -1994,6 +1994,8 @@ async def test_set_wallet_resync_on_startup(wallet_rpc_environment: WalletRpcTes
     env: WalletRpcTestEnvironment = wallet_rpc_environment
     full_node_api: FullNodeSimulator = env.full_node.api
     client: WalletRpcClient = env.wallet_1.rpc_client
+    wallet_node: WalletNode = env.wallet_1.node
+    wallet_node_2: WalletNode = env.wallet_2.node
```

Comment:
Why was this moved?

Reply:
I didn't change this. it should be because the base branch changed. I can change it back